IOS/SwiftSeedProject/SwiftSeedProject/Domain/Services/PersonService.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  PersonService.swift
+//  SwiftSeedProject
+//
+//  Created by Mugu on 3/28/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+open class PersonService {
+    let dataProvider : InMemoryDataProvider
+    
+    var persons = [Person]()
+    
+    init() {
+        dataProvider = InMemoryDataProvider()
+        let result : [AnyObject] = dataProvider.getEntities()
+        
+        for object in result {
+            let person = Person(dataModel: object as! PersonDataModel)
```

Comment:
Casting here could (and should) be avoided if the change suggested on [this ](https://github.com/makingsensetraining/mobile-pocs/pull/13/files#r109415104)comment is implemented.